packages/api/src/platforms/vtex/utils/facets.ts

```diff
@@ -27,27 +26,12 @@ export const FACET_CROSS_SELLING_MAP = {
  * */
 export const transformSelectedFacet = ({ key, value }: SelectedFacet) => {
   switch (key) {
-    case 'channel': {
-      const channel = ChannelMarshal.parse(value)
-      const channelFacets = [
-        { key: 'trade-policy', value: channel.salesChannel },
-      ]
-
-      if (channel.regionId) {
-        channelFacets.push({ key: 'region-id', value: channel.regionId })
-      }
-
-      return channelFacets
-    }
```

Comment:
Also, I didn't get why this was removed.

Reply:
this is because we were already using these values on the IS client, so we didn't really need this anymore. We were doing this computation twice